Move the content of this folder to the configured report root/js/bigipreport.js

```diff
@@ -1032,6 +1032,15 @@ function setupVirtualServerTable() {
                         result += "Yes";
                     }
                 }
+                if (type == 'filter') {
+                    if (row && row.sslprofileclient && row.sslprofileclient != "None") {
+                        result += ' ' + row.sslprofileclient;
+                    }
+                    if (row && row.sslprofileserver && row.sslprofileserver != "None") {
+                        result += ' ' + row.sslprofileserver;
+                    }
+                    console.log(result);
```

Comment:
Is the console statement intentional?

I won't be able to check this out until Sunday/Monday but figured I'd give that feedback now at least. :)

Reply:
nope, forgot to delete it. fixed.
To test, enter an ssl client or server profile name either in the main search box, or in the very small SSL heading search.

Reply:
Nice. Approved. Also will add it to the tips and tricks section of the manual.